tracker/templates/base.html

```diff
@@ -47,5 +47,29 @@
 			{%- endwith -%}
 			{%- block content -%}{%- endblock %}
 		</div>
+		<div id="page_footer">
+        	<p>
+				Copyright © 2002-2021 
```

Comment:
Ideally, we don't have to update this every year manually in archweb we have something like:
```
templates/base.html:            <p>Copyright © 2002-{% now "Y" %} <a href="mailto:jvinet@zeroflux.org"
```

So a jinja template variable should contain the current year.